### PR TITLE
Sebastianmaj/masteronlybump

### PR DESCRIPTION
**Pull request by SebastianMaj**
**Why**
Version mismatch with master and bumpconfig

**How**
Updated version in bumpconfig

### CODE REVIEW

**Comment by marvinmarnold** [comment]
@SebastianMaj have you been able to test this on a sample project? The new logic in this PR around commit messages having special meaning should at least be documented. But once we start adding special behavior like this, @shawaj's git-semantic-versioning [recommendation](https://nebraltd.slack.com/archives/C024BNQ1Y6T/p1636403605364800?thread_ts=1636398476.357100&cid=C024BNQ1Y6T) prob deserves another consideration. 

That said I still think we should get this over the line whichever the way is fastest and involves least developer friction. Is there a follow-up issue to track a more full featured solution?

**Comment by SebastianMaj** [comment]
> @SebastianMaj have you been able to test this on a sample project? The new logic in this PR around commit messages having special meaning should at least be documented. But once we start adding special behavior like this, @shawaj's git-semantic-versioning [recommendation](https://nebraltd.slack.com/archives/C024BNQ1Y6T/p1636403605364800?thread_ts=1636398476.357100&cid=C024BNQ1Y6T) prob deserves another consideration.
> 
> That said I still think we should get this over the line whichever the way is fastest and involves least developer friction. Is there a follow-up issue to track a more full featured solution?

https://github.com/SebastianMaj/hm-pytester

I had issues with it yesterday but got it working in the end in a sample project, I pushed the working workflow. I did find that it requires a Personal Access Token to have the Mr Bump commit trigger the CI workflow to successfully release. I'll see if there is a way around this but all solutions that involve a push through the workflow seemed to require this. Reason being is that SECRETS.GIT_TOKEN has an intentional blocker from running GitHub workflows to prevent an infinite loop, I made sure there is no infinite loop. 

https://github.com/NebraLtd/hm-pyhelper/issues/20

File: .github/workflows/publish-to-pypi-test.yml [hunk 1]
```diff
@@ -113,25 +117,54 @@ jobs:
...
+        with:
+          python-version: 3.7
+      
+      - name: Install pypa/build
+        run: |
+          python -m pip install build --user
+
+      - name: Assign Git User Email
+        run: |
+          git config --global user.email "mrbump@nebra.com"
```
Comment (marvinmarnold) [inline .github/workflows/publish-to-pypi-test.yml:146]: Surprised we need to do this config, don't see anything similar in hm-miner: https://github.com/NebraLtd/hm-miner/blob/master/.github/workflows/check-latest-release.yml

**Review by marvinmarnold — APPROVED** [review]
Is there a follow-up ticket that summarizes the follow-up work we want to do?

**Comment by SebastianMaj** [comment]
> Is there a ticket that summarizes the follow-up work we want to do?

Not at the moment, I can create one and include what we mentioned in todays standup

File: .github/workflows/publish-to-pypi-test.yml (hunk 1 above)
Comment (SebastianMaj) [reply .github/workflows/publish-to-pypi-test.yml:146]: This is for the commit that Mr. Bump is doing, it gave me an error the first time I implemented him saying unknown user and to set the name + email

**Comment by marvinmarnold** [comment]
Heads up @SebastianMaj https://github.com/NebraLtd/hm-pyhelper/runs/4179374630?check_suite_focus=true

**Comment by SebastianMaj** [comment]
> Heads up @SebastianMaj https://github.com/NebraLtd/hm-pyhelper/runs/4179374630?check_suite_focus=true

Oh no we can't push to master, this means we'll have to do it some other way or create a PR instead of a push. Master is a protected branch. I'll look into this either later today or on the weekend as I'll be focusing on the hm-diag exception issue.

**Pull request closed without merging**